pkg/kubelet/dockertools/manager.go

```diff
@@ -911,3 +902,35 @@ func (dm *DockerManager) KillContainer(containerID types.UID) error {
 	}
 	return err
 }
+
+// Run a single container from a pod. Returns the docker container ID
+func (dm *DockerManager) RunContainer(pod *api.Pod, container *api.Container, generator kubecontainer.RunContainerOptionsGenerator, runner kubecontainer.HandlerRunner, netMode, ipcMode string) (DockerID, error) {
```

Comment:
Naming is hard.

On Thu, Apr 23, 2015 at 10:02 PM, Yifan Gu <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7271#discussion_r29019328>
> :
>
> > @@ -911,3 +902,35 @@ func (dm *DockerManager) KillContainer(containerID types.UID) error {
> >  	}
> >  	return err
> >  }
> > +
> > +// Run a single container from a pod. Returns the docker container ID
> > +func (dm *DockerManager) RunContainer(pod *api.Pod, container *api.Container, generator kubecontainer.RunContainerOptionsGenerator, runner kubecontainer.HandlerRunner, netMode, ipcMode string) (DockerID, error) {
>
> Sorry :( I am bad at naming..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7271/files#r29019328>
> .
>